### ISSUE TITLE
Error while running ESLint: Unexpected Token, expected "," + Adjacent JSX elements must be wrapped in enclosign tag

### ISSUE THREAD

**Issue by danchristian** [vuejs/eslint-plugin-vue#479]
**Tell us about your environment**

* **ESLint Version: 4.19.1**
* **eslint-plugin-vue Version: 4.5.0**
* **Node Version: 8.11.1**

**Please show your full configuration:**

```
module.exports = {
  root: true,
  env: {
    browser: true,
    node: true,
    es6: true
  },
  plugins: ["vue", "prettier"],
  extends: [
    "eslint:recommended",
    "plugin:vue/recommended",
    "plugin:prettier/recommended"
  ],
  rules: {
    "prettier/prettier": "error"
  },
  parserOptions: {
    sourceType: "module"
  }
};
```

**What did you do? Please include the actual source code causing the issue.**

```vue
<template lang="pug">
  ul
    li(v-for="withdrawal in bankingWithdrawals") {{ withdrawal.Amount }}
</template>
<script>
</script>
```

**What did you expect to happen?**

No errors

**What actually happened? Please include the actual, raw output from ESLint.**

It won't allow periods in mustache syntax, it keeps on erroring. I can't find any related issues on this repo.

<img width="844" alt="screen shot 2018-05-25 at 11 26 58" src="https://user-images.githubusercontent.com/5786541/40540028-95c7483e-600e-11e8-8c55-d413e22fb02d.png">

If I remove `{{ withdrawal.Amount }}` and reload, I get a new error, complaining about adjacent JSX elements. I am not using JSX and the other issues on this repo similar to this don't solve my problem.

<img width="951" alt="screen shot 2018-05-25 at 11 30 02" src="https://user-images.githubusercontent.com/5786541/40540137-fe1eeb9e-600e-11e8-9854-52f6b0cc7a25.png">
